### PR TITLE
Fixed stucked notification

### LINKED ISSUE #66: Stop recording location does not remove the notification bar
When the location logging is not on, even then the notification bar keeps showing the location is getting recorded.

### PR DESCRIPTION
**Pull request by HrushikeshAgrawal**
Fixes #66

### CODE REVIEW

File: app/services/LocationService.js [hunk 1]
```diff
@@ -1,270 +1,295 @@
...
+  static stop() {
+    // unregister all event listeners
+    BackgroundGeolocation.removeAllListeners();
+    BackgroundGeolocation.stop();
+    instanceCount -= 1;
+  }
+
+  static optOut(nav) {
+    BackgroundGeolocation.removeAllListeners();
+    BackgroundGeolocation.stop();
```
Comment (tremblerz) [inline app/services/LocationService.js]: This is the only code change right? just want to make sure that I am not missing any extra piece of code which has been added/removed.
Also, if possible please apply changes in such a way that code difference is minimized, right now it is hard to navigate across.

Comment (HrushikeshAgrawal) [reply app/services/LocationService.js]: Yes, that (BackgroundGeolocation.stop();) is the only code change I had made. The rest of the changes were due to the code formatter I used while making the change. Will be careful from next time and not use the code formatter. So, would you like me to close this pull request and create a new one or should we continue with this one?

Comment (rparet) [reply app/services/LocationService.js]: Hi there. Let's continue with this one. Please keep this PR open but push up a new commit that is only your change, and no code formatter changes. Thanks!

**Comment by rparet** [comment]
Fixed in #115 - thank you very much!

**Pull request closed without merging**